Code/Framework/AzFramework/Platform/Linux/platform_linux.cmake

```diff
@@ -10,17 +10,12 @@
 # Only 'xcb' and 'wayland' are recognized
 if (${PAL_TRAIT_LINUX_WINDOW_MANAGER} STREQUAL "xcb")
 
-    find_library(XCB_LIBRARY xcb)
-    find_library(XCB_XKB_LIBRARY xcb-xkb)
-    find_library(XKBCOMMON_LIBRARY xkbcommon)
-    find_library(XKBCOMMON_X11_LIBRARY xkbcommon-x11)
-
     set(LY_BUILD_DEPENDENCIES
         PRIVATE
-            ${XCB_LIBRARY}
-            ${XKBCOMMON_LIBRARY}
-            ${XKBCOMMON_X11_LIBRARY}
-            ${XCB_XKB_LIBRARY}
+            3rdParty::X11Interface::xcb
+            3rdParty::X11Interface::xcb_xkb
+            3rdParty::X11Interface::xkbcommon
+            3rdParty::X11Interface::xkbcommon_X11
```

Comment:
So for a discussion point.
I don't think `3rdParty` is quite right for the `namespacing` of targets such as these.
Since these are libraries that come from the system itself, how about having the namespace be `System`

Like `System::X11Interface::xcb`

Reply:
I'm open to that idea, but I think we'd have to extend a bunch of our `if(name STREQUAL 3rdParty` checks to include `OR name STREQUAL System` to make that work. Is it worth it?

Reply:
Why do we even check, though?